client/components/payment-activity/index.tsx

```diff
@@ -85,9 +79,14 @@ const PaymentActivity: React.FC = () => {
 
 	return (
 		<Card>
-			<CardHeader>
+			<CardHeader className="wcpay-payment-activity__card__header">
 				{ __( 'Your payment activity', 'woocommerce-payments' ) }
-				{ /* Filters go here */ }
+				<DateRangePicker
+					selectedKey={ dateRange.key }
```

Comment:
As above, suggest using a clearer name that is merchant focused, e.g. `preset_name` or `period` or similar.

Reply:
This has been renamed to `preset_name`. Marking this issue as resolved.